packages/client/src/views/RegisterForm/review/ReviewSection.tsx

```diff
@@ -653,23 +647,18 @@ class ReviewSectionComp extends React.Component<FullProps, State> {
 
     uploadedDocuments = uploadedDocuments.filter((document) => {
       const sectionMapping = SECTION_MAPPING[draft.event]
-      const sectionTitle = SECTION_TITLE[draft.event]
 
       const allowedDocumentType: string[] =
         sectionMapping[activeSection as keyof typeof sectionMapping] || []
 
       if (
         allowedDocumentType.indexOf(document.optionValues[0]!.toString()) > -1
       ) {
-        const title: string =
-          sectionTitle[activeSection as keyof typeof sectionMapping]
         const label =
-          intl.formatMessage(messages.documentForWhom, {
-            section: title.toLowerCase()
-          }) +
-          ' ' +
-          (this.getLabelForDocType(title, document.optionValues[1] as string) ||
-            document.optionValues[1])
+          this.getLabelForDoc(
+            document.optionValues[0] as string,
```

Comment:
why does this need to be cast as a string? can it be something other than a string